src/constants/errors.json

```diff
@@ -17,6 +18,7 @@
     "VALIDATION_FAILED": "Validation Failed",
     "DUPLICATE_KEY": "Data violates unique constraints validation",
     "INVALID_ID": "Invalid id",
+    "BAD_LOCATION": "The location co-ordinates do not follow the correct format: [Longitude, Latitude]",
```

Comment:
add the range for each value or is it not necessary (in the docs as well maybe)?